Extension/src/main.ts

```diff
@@ -138,27 +138,30 @@ export function activate(context: vscode.ExtensionContext): void {
     registerTempCommand("C_Cpp.TakeSurvey");
 
     processRuntimeDependencies(() => {
-        downloadCpptoolsJsonPkg().then(() => {
-            util.readFileText(util.getExtensionFilePath("cpptools.json"))
-                .then((cpptoolsString) => {
-                    processCpptoolsJson(cpptoolsString);
-                })
-                .catch((error) => {
-                    // We already log telemetry if cpptools.json fails to download.
-                })
-                .then(() => {
-                    // Main activation code.
-                    tempCommands.forEach((command) => {
-                        command.dispose();
-                    });
-                    tempCommands = [];
-                    LanguageServer.activate(delayedCommandsToExecute);
-                    delayedCommandsToExecute.forEach((command) => {
-                        vscode.commands.executeCommand(command);
-                    });
-                    delayedCommandsToExecute.clear();
+        util.readFileText(util.getExtensionFilePath("cpptools.json"))
+            .then((cpptoolsString) => {
+                processCpptoolsJson(cpptoolsString);
+            })
+            .catch((error) => {
+                // We already log telemetry if cpptools.json fails to download.
```

Comment:
minor: update comment.  This code path does not download anything anymore.

Reply:
I don't understand. What code path?

Reply:
I take it back.  I made this comment through the web UI and didn't have the full context.